src/components/SignupButton.vue

```diff
@@ -14,7 +14,9 @@ export default {
   methods: {
     signup() {
       this.$auth.loginWithRedirect({
-        screen_hint: "signup",
+        authorizationParams: {
+          screen_hint: "signup",
+        },
```

Comment:
We introduced `authorizationParams`: https://github.com/auth0/auth0-spa-js/blob/master/MIGRATION_GUIDE.md#introduction-of-authorizationparams